PokemonSearch/PokemonSearch/SearchTableViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SearchTableViewController.swift
+//  PokemonSearch
+//
+//  Created by Jocelyn Stuart on 1/25/19.
+//  Copyright © 2019 JS. All rights reserved.
+//
+
+import UIKit
+
+class SearchTableViewController: UITableViewController, UISearchBarDelegate {
+
+    var pokemonController = PokemonController()
+    
+    var pokemon: Pokemon?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+        guard let searchTerm = searchBar.text, !searchTerm.isEmpty else { return }
+        
+        pokemonController.searchForPokemon(with: searchTerm.lowercased()) { (pokemon, error) in
+            DispatchQueue.main.async {
+                self.pokemon = pokemon
+                self.tableView.reloadData()
+            }
+        }
+    }
+    
+    // MARK: - Table view data source
+
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return pokemons.count
```

Comment:
When performing a search we can't see any result. Because you're using wrong array as a data source. You have created this array only for saved pokemon and should be used in `PokemonTableViewController` as data source. Your search is going to be return just one result for now, and the table view needs just one row
```suggestion
        return 1
```